docs/source/jsonpath.rst

```diff
@@ -7,10 +7,14 @@ validating response bodies that are formatted as JSON and making
 reference to that JSON in subsequent queries. `jsonpath_rw`_ is used
 to process the JSONPath expressions.
 
-To address a common requirement when evaluting JSON responses, an
-extension has been made to the default implementation of JSONPath.
-This extension is ``len`` and will return the length of the current
-datum in the JSONPath expression.
+To address common requirements when evaluting JSON responses,
+extensions have been made to the default implementation of JSONPath
+using `jsonpath_rw_ext`. It adds three useful functions:
+
+#. Return the length of the current datum using ``len``.
+#. Sort the current datum using ``sorted`` and ``[/name]``.
```

Comment:
FWIW, I only understood this by looking at the tests. (I'm aware of https://github.com/cdent/gabbi/pull/115#discussion_r55124235 though.)